commcare_export/builtin_queries.py

```diff
@@ -22,22 +23,27 @@ def mapped_source_field(self):
         if not self.map_function:
             return Reference(self.source)
         else:
-            return Apply(Reference(self.map_function), Reference(self.source),
-                         *self.extra_args)
+            return Apply(
+                Reference(self.map_function), Reference(self.source),
+                *self.extra_args
+            )
 
 
 def compile_query(columns, data_source, table_name):
-    source = Apply(Reference('api_data'), Literal(data_source),
-                   Reference('checkpoint_manager'))
-    part = excel_query.SheetParts(table_name, [c.name for c in columns], source,
-                                  List([c.mapped_source_field for c in columns]),
-                                  None)
+    source = Apply(
+        Reference('api_data'), Literal(data_source),
+        Reference('checkpoint_manager')
```

Comment:
This one is a bit annoying. Wonder why it didn't do
```suggestion
        Reference('api_data'),
        Literal(data_source),
        Reference('checkpoint_manager')
```